config/esp32/components/chip/Kconfig

```diff
@@ -768,4 +768,20 @@ menu "CHIP Device Layer"
 
     endmenu
 
+    menu "Matter OTA Image"
+
+        config CHIP_OTA_IMAGE_BUILD
+            bool "Generate Matter OTA image"
```

Comment:
We should also update our examples documentation that use OTA to mention how to create the ota image (script, command)